examples/dockerfiles/README.md

```diff
@@ -0,0 +1,28 @@
+# Instructions for using the Dockerfiles
+
+To build these dockerfiles go to the Gluon-TS root directory and run:
+
+```bash 
+docker build . -f examples/dockerfiles/<the dockerfile>
+```
+
+To run the built image run:
+
+```bash
+docker run <image_id> <shell_params>
```

Comment:
Let's maybe leave this out, and just say that these images can be used on SageMaker?

Reply:
Removed this